flows/actions/testdata/send_msg.json

```diff
@@ -485,6 +486,7 @@
                             "name": "affirmation"
                         },
                         "language": "spa",
+                        "country": "",
```

Comment:
I assume not all templates have country set

Reply:
In the whatsapp world they always will but ya, generally no.